include/oneapi/dpl/internal/dynamic_selection_traits.h

```diff
@@ -13,6 +13,8 @@
 #include <utility>
 #include <cstdint>
 #include <type_traits>
+#include <thread>
+#include <optional>
```

Comment:
```suggestion
```

Reply:
Removed this and other similar includes of `optional`